deployment/helm/human-connection/values.yaml

```diff
@@ -1,31 +1,27 @@
-domain: nitro-staging.human-connection.org
+domain: develop.human-connection.org
 commit: 889a7cdd24dda04a139b2b77d626e984d6db6781
 pullPolicy: Always
 
 # configs
-developmentMailserverDomain: "helm-nitro-mailserver.human-connection.org"
-humanConnectionDomain: "helm-staging.human-connection.org"
-supportEmail: "developer@human-connection.org"
-letsencryptIssuer: "letsencrypt-staging"
+developmentMailserverDomain: "mailserver.human-connection.org"
+humanConnectionDomain: "develop.human-connection.org"
+supportEmail: "devcom@human-connection.org"
```

Comment:
Do we have that email? :thinking: Or just to avoid spam?

Reply:
this is the email that @datenbrei asked to add, but it goes back to the point I made in `Discord` that if we are going to publish our `helm` chart publicly, it would be nice to get set up with `helm-secrets` and encrypt the files.